style.css

```diff
@@ -153,3 +159,378 @@ header {
+}
+
+.right-block {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 0;
+  gap: 12px;
+  width: 327px;
+  height: 305px;
+}
+
+.block-h2 {
+  width: 327px;
+  height: 36px;
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  color: #091e42;
+  margin: 0;
+}
+
+.block-p {
+  width: 327px;
+  height: 147px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #091e42;
+  margin: 0;
+}
+
+.block-nav > ul {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  margin: 0;
+  padding: 0;
+  gap: 12px;
+  width: 297px;
+  height: 40px;
+}
+
+.nav-language {
+  width: 25px;
+  height: 20px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+}
+
+.nav-list {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 10px 12px;
+  width: 56px;
+  height: 40px;
+  background: #fff;
+  border: 1px solid #8993a4;
+}
+
+#bootstrap {
+  flex-grow: 2;
+}
+
+.right-block > button {
+  width: 120px;
+  height: 40px;
+  background: #ff6b00;
+  border: none;
+  margin: 0;
+  padding: 10px;
+  color: #fff;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+}
+
+.frame43 {
+  display: grid;
+  grid-template-rows: repeat(6, 1fr);
+  padding: 0 12px 0 0;
+  gap: 24px;
+  width: 327px;
+  height: 2550px;
+}
+
+.card-work {
+  width: 327px;
+  height: 386px;
+  background-image: url(images/cardwork.svg);
+  display: grid;
+  grid-auto-rows: 4fr 1fr;
+}
+
+.group {
+  display: grid;
+  grid-template-rows: 1fr 1fr 0.5fr;
+}
+
+.group-header {
+  width: 295px;
+  height: 81px;
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  color: #fff;
+}
+
+.group-para {
+  width: 295px;
+  height: 87px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #fff;
+}
+
+.group-ul {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 0;
+  gap: 12px;
+  width: 236px;
+  height: 40px;
+}
+
+.group-ul > li {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 10px 12px;
+  width: 56px;
+  height: 40px;
+  background: rgba(255, 255, 255, 0.24);
+}
+
+.bootstrap {
+  flex-grow: 2;
+}
+
+.group-a {
+  width: 32px;
+  height: 20px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: #fff;
+}
+
+.action {
+  height: 48px;
+  background: #ff6b00;
+  color: #fff;
+  letter-spacing: 0.03em;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  border: none;
+}
```

Comment:
- [ ] Please, consider adding all the interaction effects to your buttons as suggested in the requirements of this milestone! 🤓 🛠️ 

- Milestone requirements:
![image](https://user-images.githubusercontent.com/77069011/199251236-df4d7eb6-1c1a-410b-96a9-85f3214ec122.png)

- Button interactions:
![Screenshot from 2022-11-01 07-32-12](https://user-images.githubusercontent.com/77069011/199251405-e9f8750e-4205-4817-8c57-67f03e453186.png)